lms/templates/fields/message_banner.underscore

```diff
@@ -1,4 +1,4 @@
-<div class="wrapper-msg urgency-<%= urgency %> <%= type %>">
+<div class="wrapper-msg urgency-<%= urgency %> <%= type %>" role="alert">
```

Comment:
adding role="alert" for this banner.  
This will also affect other banners using this template.  
This would probably count as a bug with all the other banners using this template previously.